[doc] Fix inconsistencies with torch.linalg.det docs

heitorschueroff opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #51672 [doc] Fix inconsistencies with torch.linalg.inv and deprecate torch.inverse
* #51671 [doc] Fix inconsistencies with linalg.pinv docs and deprecate pinverse
* #51660 doc] Fix inconsistencies with torch.linalg.matrix_rank doc
* #51659 [doc] Fix inconsistencies with torch.linalg.eigvalsh
* #51658 [doc] Fix inconsistencies with torch.linalg.eigh
* **#51651 [doc] Fix inconsistencies with torch.linalg.det docs**
* #51641 [doc] Fix inconsistencies with torch.linalg.cond doc

Differential Revision: [D26234103](https://our.internmc.facebook.com/intern/diff/D26234103)

heitorschueroff committed:
[doc] Fix inconsistencies with torch.linalg.det docs

[ghstack-poisoned]

heitorschueroff committed:
Update on "[doc] Fix inconsistencies with torch.linalg.det docs"




[ghstack-poisoned]

mruberry commented on torch/linalg/__init__.py:
```diff
@@ -145,7 +145,46 @@
 linalg.det(input) -> Tensor
 
-Alias of :func:`torch.det`.
+Computes the determinant of a square matrix or batch of square matrices.
```
Tweak this language and give the HTML a last sanity check. Overall this looks great!

heitorschueroff committed:
Update on "[doc] Fix inconsistencies with torch.linalg.det docs"


Differential Revision: [D26234103](https://our.internmc.facebook.com/intern/diff/D26234103)

[ghstack-poisoned]

heitorschueroff committed:
Update on "[doc] Fix inconsistencies with torch.linalg.det docs"


Differential Revision: [D26234103](https://our.internmc.facebook.com/intern/diff/D26234103)

[ghstack-poisoned]

heitorschueroff committed:
Update on "[doc] Fix inconsistencies with torch.linalg.det docs"


Differential Revision: [D26234103](https://our.internmc.facebook.com/intern/diff/D26234103)

[ghstack-poisoned]

heitorschueroff committed:
Update on "[doc] Fix inconsistencies with torch.linalg.det docs"


Differential Revision: [D26234103](https://our.internmc.facebook.com/intern/diff/D26234103)

[ghstack-poisoned]

heitorschueroff committed:
Update on "[doc] Fix inconsistencies with torch.linalg.det docs"


Differential Revision: [D26234103](https://our.internmc.facebook.com/intern/diff/D26234103)

[ghstack-poisoned]
